packages/suite/src/views/wallet/transactions/TradeBox/TradeBoxGraph.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTheme } from '@trezor/components';
+import { Line, LineChart } from 'recharts';
+
+export interface TradeBoxGraphProps {
+    tickers: { ts: number; rate: number }[];
+}
+
+function normalizeRate(val: number, min: number, max: number) {
+    return (val - min) / (max - min);
+}
+
+export const TradeBoxGraph = ({ tickers }: TradeBoxGraphProps) => {
+    const theme = useTheme();
```

Comment:
use the one from `styled-components` please, the one in `@trezor/components` is gonna be deprecated soon :)